platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -244,4 +270,41 @@ private String getTruststorePassword() {
   List<Validator> getValidators() {
     return validators;
   }
+
+  private SystemNotice createUsersPropertiesNotice() {
+    Instant firstInstall = scheduler.installationDate();
+    Set<String> details = new HashSet<>();
+    String title;
+
+    if (firstInstall != null) {
+      Instant timestamp = firstInstall.plus(Duration.ofDays(3));
+
+      DateTimeFormatter formatter =
+          DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM)
+              .withLocale(Locale.getDefault())
+              .withZone(ZoneId.systemDefault());
+      String timestampString = formatter.format(timestamp);
+      title =
+          String.format(
+              "USERS.PROPERTIES FILE FOUND! Follow the instructions below before %s to prevent the system from locking down.",
+              timestampString);
+      details.add(
+          String.format(
+              "%s It will be automatically deleted on %s. %s",
+              FILE_FOUND_MSG, timestampString, FOLLOW_DOC_MSG));
+
+    } else {
```

Comment:
Do we not always have an install date? If we don't should we create it before we get to this point?

Reply:
We should I was trying to avoid if for some reason it fails to post a message like `It will automatically be deleted on null`.